tutorials/core-concepts/users-and-permissions/managing-accesss-control.md

```diff
@@ -1,5 +1,159 @@
 ---
-title: Manaing Access Control
+title: Managing Access Control
 layout: tutorial
 ---
 
+# Managing Access Control
+
+In most cases, you essentially have three levels of permissions:
+
+* The administrator (`admin`), who can do anything
+
+* The editor or writer (`edit`), who can create content, but doesn't have access to configuration or management of the site
```

Comment:
... who can create content of all piece types (other than users and groups) and edit pages where specifically granted permission to do so

Reply:
Updated.